pkg/apiserver/diagnose/query.go

```diff
@@ -118,37 +152,65 @@ func (t sumValueQuery) queryRow(arg *queryArg, db *gorm.DB) (*TableRowDef, error
 	if len(t.name) == 0 {
 		t.name = t.tbl
 	}
-	condition := fmt.Sprintf("where time >= '%s' and time < '%s' ", arg.startTime, arg.endTime)
+	fields := fmt.Sprintf(`
+		'%s' as name, 
+		'' as label,
+		sum(value) as sum
+		`, t.name)
+	table := fmt.Sprintf("metrics_schema.%s", t.tbl)
+
+	query := db.
+		Select(fields).
+		Table(table).
+		Where("time >= ? AND time < ?", arg.startTime, arg.endTime)
 	if len(t.condition) > 0 {
-		condition = condition + "and " + t.condition
+		query = query.Where(t.condition)
+	}
+	rows, err := query.Rows()
+	if err != nil {
+		return nil, err
 	}
-	sql := fmt.Sprintf("select '%s', '', sum(value) from metrics_schema.%s %s",
-		t.name, t.tbl, condition)
-	rows, err := querySQL(db, sql)
+	results, err := scanRows(rows)
 	if err != nil {
 		return nil, err
 	}
-	if len(rows) == 0 {
+	if len(results) == 0 {
 		return nil, nil
 	}
 	if len(t.labels) == 0 {
-		return t.genRow(rows[0], nil), nil
+		return t.genRow(results[0], nil), nil
+	}
+
+	labelFields := fmt.Sprintf("`%v`", strings.Join(t.labels, "`||','||`"))
```

Comment:
the original SQL is assembled using `strings.Join(t.labels, "`,`")`

Reply:
`||` is used as a connector in `gorm`, equivalent to concatenating strings